doctr/transforms/functional/pytorch.py

```diff
@@ -32,7 +32,7 @@ def rotate(
     img: torch.Tensor,
     boxes: np.ndarray,
     angle: float,
-    expand: bool = False,
+    expand: bool = True,
```

Comment:
I would advise against it: I didn't enable it on purpose in #438.
This has consequences on predictors which we still need to investigate carefully as mentioned in #225